### DISCUSSION TITLE
Can I create my own data type?

### DISCUSSION THREAD

**Discussion by oleksandr-petryk** [prisma/prisma#12782] [Q&A] [answered]
I want to store [`TIME WITH TIMEZONE`](https://www.postgresql.org/docs/9.1/datatype-datetime.html) in PostgreSQL, there are a way to implement it?

**Comment by austincrim** [comment, marked as answer]
Hey @Sasha-hk 👋,

You can use a [native Postgres type mapping](https://www.prisma.io/docs/concepts/database-connectors/postgresql#native-type-mappings) for this, specifically `@db.Timetz()`.

```prisma
sometime  DateTime @db.Timetz()
```

Let me know if that works!

**Reply by oleksandr-petryk** [reply]
That's what I need, thank you

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
